### ISSUE TITLE
java.lang.UnsatisfiedLinkError

### ISSUE THREAD

**Issue by Bl4Cc4t** [kptdobe/k8055#1]
Hey,
I’m very happy that you made this project. However, it is not really working for me...
I’m currently using Eclipse as IDE and get this error when running from `Tester.java`:
```
Exception in thread "main" java.lang.UnsatisfiedLinkError: k8055.K8055Interface.OpenDevice(I)I
	at k8055.K8055Interface.OpenDevice(Native Method)
	at k8055.K8055Interface.open(K8055Interface.java:56)
	at k8055.Tester.main(Tester.java:25)
```

I’ve put `libk8055.dylib` in the folder `~/Library/Java/Extensions` and installed the kext from [piha](https://github.com/piha/k8055-mac-codeless-kext), like you linked.
Did I miss something?

**Comment by kptdobe** [comment]
Really old code ;)
It is a path issue, it cannot find the library.
Is your `~/Library/Java/Extensions` path in the library path ? something like -`Djava.library.path="~/Library/Java/Extensions"`
Stackoverflow should be your friend: https://stackoverflow.com/questions/957700/how-to-set-the-java-library-path-from-eclipse

**Comment by Bl4Cc4t** [comment]
Well it outputs this path when I ran `Tester.java`, so put it in there.
Just modified the settings as described [here](https://stackoverflow.com/a/41741940/7889988), still the same error. It even prints me my new path, in which I put the dylib...

On another note, I tried the same thing on a friends laptop. On his side, Eclipse did not complain. 
But there was no communication happening; the board displayed nothing... Tried nearly all the methods, no luck...

I hope I’ll be able to fix this import error somehow. Maybe trying an older version of Eclipse will do something? I’ll try later, will edit this.

Edit: Tried it with Mars, same error. Even rebooted >_<
